.github/workflows/auto_translate.yml

```diff
@@ -0,0 +1,29 @@
+name: Auto translate
+on:
+  workflow_dispatch:
+
+jobs:
+  autotranslate:
+    env:
+      API_KEY: ${{secrets.DL_API_KEY}}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
```suggestion
      - uses: actions/checkout@v4
```

Reply:
On my phone. Make whatever changes that are needed

Reply:
> use one of the shared automations at OpenVoiceOS/.github

There are none. we should invest time into setting them up

Reply:
I am not versed well enough for the automatons yet, shared ones would be great though.  Much easier to setup